instagram-app/src/App.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import PostPage from './components/PostContainer/PostPage';
+import LoginPage from './components/LoginPage/LoginPage'
+import withAuthentication from './components/Authentication/Authentication';
+
+
+class App extends Component {
+  constructor() {
+    console.log('Constructing....')
```

Comment:
Make sure you remove your `console.logs` when you commit your code. This will keep your code nice and tidy. 👍